locales/ru_RU/pcsx2_Iconized.po

```diff
@@ -33,61 +33,59 @@ msgstr ""
 msgid ""
 "Failed to read/write NVM/MEC file. Check your BIOS setup/permission settings."
 msgstr ""
-"Не удалось прочитать/записать файл NVM/MEC. Проверьте настройки BIOS/"
-"настройки разрешений."
+"Не удалось прочитать/записать файл NVM/MEC. Проверьте настройки BIOS/настройки "
+"разрешений."
 
 #: pcsx2/System.cpp:112
 msgid ""
-"This recompiler was unable to reserve contiguous memory required for "
-"internal caches.  This error can be caused by low virtual memory resources, "
-"such as a small or disabled swapfile, or by another program that is hogging "
-"a lot of memory."
+"This recompiler was unable to reserve contiguous memory required for internal "
```

Comment:
Are the English string changes necessary?

Reply:
no, Poedit editor for some reason changed them when saving

Reply:
Fixed

Reply:
There are other strings too btw.